packages/tests/components/SfRatingButton/SfRatingButton.cy.tsx

```diff
@@ -1,39 +1,29 @@
 /// <reference path="../../../../node_modules/@percy/cypress/types/index.d.ts" />
 import React from 'react';
-import { SfRatingButtonSize, noop } from '@storefront-ui/shared';
+import { SfRatingButtonSize } from '@storefront-ui/shared';
 import { mount, useComponent } from '../../utils/mount';
 
 const { vue: SfRatingButtonVue, react: SfRatingButtonReact } = useComponent('SfRatingButton');
 import SfRatingButtonBaseObject from './SfRatingButton.PageObject';
 import { isReact } from '../../utils/utils';
 
+interface Props {
+  value?: number;
+  onChange?: (value: number) => void;
+  max?: number;
+  name?: string;
+  disabled?: boolean;
+  label?: string;
+  labelClassName?: string;
+  size?: `${SfRatingButtonSize}`;
+  getLabelText?: (value: number) => string;
+}
+
```

Comment:
question: those are copy paste from https://github.com/vuestorefront/storefront-ui/pull/2820/files#diff-8fcdae6ac93bc10b998e13c1280c073dd3e3ff2cac241eafea1e74d5ebe84ca0R13 why not reuse those ?